diesel_cli/src/cli.rs

```diff
@@ -121,6 +121,10 @@ pub fn build_cli() -> App<'static, 'static> {
                         .help("The format of the migration to be generated."),
                 ),
         )
+        .subcommand(
+            SubCommand::with_name("features")
+                .about("List which features have been compiled into this version of diesel_cli"),
+        )
```

Comment:
This change is meaningless. You already added the subcommand below.

Reply:
I will remove this.